src/absences/types.ts

```diff
@@ -55,7 +55,7 @@ export type Absence = {
   updated: string;
 };
 
-export type AbsenceCreate = Omit<Absence, 'id' | 'created' | 'status' | 'duration_unit' | 'updated'>;
+export type AbsenceCreate = Omit<Absence, 'id' | 'created' | 'status' | 'duration_unit' | 'updated' | 'duration'> & { duration?: number };
```

Comment:
Some other option is also:
`export type AbsenceCreate = Omit<Absence, 'id' | 'created' | 'status' | 'duration_unit' | 'updated' | 'duration'> & Partial<Pick<Absence, 'duration>>;`